packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -158,6 +159,7 @@ export default [
             ...getNewSettings(state.newSchedules),
             '': {
               reportWhen: state.reportWhen,
+              concurrency: state.concurrency || undefined,
```

Comment:
What's the point of this?

Reply:
The concurrency's default value equals 0 in the UI.